### DISCUSSION TITLE
Getting the selected  `Run and Debug` configuration

### DISCUSSION THREAD

**Discussion by OrkunTokdemir** [microsoft/vscode-discussions#1116] [Extension Development QnA] [answered]
I'm working on the qt vscode extension. I need to get the `workspaceFolder` value of the selected debug config. When I checked the documentation, I noticed that only the `activeDebugSession`  configuration can be obtained. But my problem is that I'm providing some debug config variables and those variables depend on the selected configuration task. Also, there is no `activeDebugSession` when my variables are triggered. So `vscode.debug.activeDebugSession` returns `undefined`.
![image](https://github.com/microsoft/vscode-discussions/assets/21346859/2b014e51-d747-424e-9abf-6473adc33219)
In the above screenshot, there are two open workspaces. Those are `qt_playground` and `qt_playground2`. How can I get the value of the selected one before activating a debug session?

**Comment by isidorn** [comment, marked as answer]
Not possible through the current API. 
Though a fair feature request. fyi @connor4312
